src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/EcsLoadBalancerV2Handler.java

```diff
@@ -131,6 +133,10 @@ public LoadBalancer createLoadBalancer(EcsClusterMetadata clusterMetadata, EcsPu
 
         List<com.amazonaws.services.elasticloadbalancingv2.model.Tag> tags = getElbTagList(
             clusterMetadata.getClusterCftStackTags(), appName);
+        tags.addAll(definition.getTags().stream().map(hermanTag ->
+            new com.amazonaws.services.elasticloadbalancingv2.model.Tag().withKey(hermanTag.getKey()).withValue(hermanTag.getValue())
```

Comment:
Don't you wish we converted Herman to Kotlin @rdbatch02 ;-)

Reply:
:sadpanda: